client/components/telecom/telephony/abbreviatedNumbers/telecom-telephony-abbreviated-numbers.modal.html

```diff
@@ -1,171 +1,77 @@
-                      data-translate="telephony_abbreviated_numbers_not_free_error">
-                </span>
-            </span>
-        </div>
+                data-ng-model="AbbreviatedNumberModal.data.abbreviatedNumber">
+        </oui-field>
 
-        <!-- Number -->
-        <div class="form-group"
-             data-ng-class="{
-                'has-error': abbreviatedNumberForm.destinationNumber.$touched && abbreviatedNumberForm.destinationNumber.$invalid
-             }">
-            <label for="destinationNumber"
-                   class="control-label"
-                   data-translate="telephony_abbreviated_numbers_number">
-            </label>
+        <oui-field
+            data-label="{{:: 'telephony_abbreviated_numbers_number' | translate }}"
+            data-error-messages="{pattern: ('telephony_abbreviated_numbers_number_error' | translate) }"
+            data-help-text="{{:: 'telephony_abbreviated_numbers_number_help' | translate }}">
             <input
                 type="text"
                 name="destinationNumber"
                 id="destinationNumber"
-                class="form-control"
-                data-translate
-                translate-attr-placeholder="telephony_abbreviated_numbers_number"
+                class="oui-input"
                 data-ng-pattern="AbbreviatedNumberModal.numberPattern"
                 required
                 data-ng-readonly="AbbreviatedNumberModal.readOnly.destinationNumber"
-                data-ng-model="AbbreviatedNumberModal.data.destinationNumber"/>
-            <span class="help-block"
-                  role="alert"
-                  data-ng-if="abbreviatedNumberForm.destinationNumber.$touched"
-                  data-ng-messages="abbreviatedNumberForm.destinationNumber.$error">
-                <span data-ng-message="pattern"
-                      data-translate="telephony_abbreviated_numbers_number_error">
-                </span>
-            </span>
-        </div>
+                data-ng-model="AbbreviatedNumberModal.data.destinationNumber">
+        </oui-field>
 
-        <!-- Surname -->
-        <div class="form-group"
-             data-ng-class="{
-                'has-error': abbreviatedNumberForm.surname.$touched && abbreviatedNumberForm.surname.$invalid
-             }">
-            <label for="surname"
-                   class="control-label"
-                   data-translate="telephony_abbreviated_numbers_surname">
-            </label>
+        <oui-field
+            data-label="{{:: 'telephony_abbreviated_numbers_surname' | translate }}">
             <input
                 type="text"
                 name="surname"
                 id="surname"
-                class="form-control"
-                data-translate
-                translate-attr-placeholder="telephony_abbreviated_numbers_surname"
-                maxlength="85"
+                class="oui-input"
+                data-ng-maxlength="85"
```

Comment:
`maxlength` should be enough here as maxlength is fixed to `85` ?

Reply:
the behavior is different maxlength vs ng-maxlength
"The ng-maxlength is not the same as the maxlength attribute in HTML, which will prevent users from typing more than the restricted number of characters.

The ng-maxlength directive will not prevent users from typing more than the restricted number off characters, but the form will be invalid if they do so."

CX want ng-maxlength behavior